bin/publish

```diff
@@ -5,8 +5,8 @@
 REDHAT_IMAGE='scan.connect.redhat.com/ospid-1c46a2de-1d88-40e6-a433-7114ad0099cb/conjur-openshift-authenticator-client'
 
 readonly REGISTRY="cyberark"
+readonly VERSION="$(short_version_tag)"
 readonly TAG="$(version_tag)"
-readonly VERSION="$(cat VERSION)"
```

Comment:
there is more work that needs to be done in this file to ensure this project is able to publish appropriately.

previously the build script tagged the images as `conjur-authn-k8s-client:dev` and `conjur-authn-k8s-client:dev-redhat`

now it is tagging them as `conjur-authn-k8s-client:v1.2.3-{hash}` and `conjur-authn-k8s-client:v1.2.3-{hash}-redhat`

later in this file (in more than one place, but see line 28 for an example) we retag the image that was built locally and publish it.

do we need to change the tag that's output in bin/build? could we also tag the image built in bin/build with `latest` and `latest-redhat` and use those tags in this script, to avoid the complexity of referring to the specific version + hash?

Reply:
If I am understanding correctly, leave the build to output `conjur-authn-k8s-client:dev` so that it is tagged in publish with the proper `latest` tag and just output in `main.go` the version `(v1.2.3-{hash})`

Reply:
is there a way to run through the publish locally to ensure it will publish in the proper format?